content/applications/hr/payroll/salary_attachments.rst

```diff
@@ -0,0 +1,151 @@
+==================
+Salary attachments
+==================
+
+Salary attachments are portions of earnings taken directly out of a payslip for a specific purpose,
+whether voluntary or required.
+
+When the deduction is voluntary, they are typically considered *deductions*. When the deduction is
+court-ordered, or involuntary, it is sometimes referred to as a *wage garnishment*. In Odoo, these
+are all universally called, *salary attachments*.
+
+.. _salary-attachment/attachment-types:
```

Comment:
Please make sure [custom anchors follow the convention](https://www.odoo.com/documentation/18.0/contributing/documentation/rst_guidelines.html#custom-anchors) :)
```suggestion
.. _payroll/salary-attachment/types:
```